content/en/os/1.13.x/api/endpoints/index.markdown

```diff
@@ -0,0 +1,11 @@
++++
+title = "API Endpoints"
+type = "swagger"
+description = "Paths and methods for the API"
++++
+
+The following output is generated from [Bottlerocket's OpenAPI Spec](https://github.com/bottlerocket-os/bottlerocket/blob/develop/sources/api/openapi.yaml).
+
+---
+
+{{<  swaggerui src="../../../../external/openapi.yaml" >}}
```

Comment:
Not sure if this is expected or not, but when I click on e.g. `GET /settings` to expand that row, then close it (not having clicked "try it out" or any buttons), I see this error at the top of the page:

<img width="767" alt="image" src="https://user-images.githubusercontent.com/10101544/230196247-c1918dad-9ad4-4d47-9898-c6d2a4b796da.png">

Reply:
No, not expected.

However, this [appears to be an issue with Swagger UI](https://stackoverflow.com/questions/65584131/weird-could-not-resolve-reference-undefined-not-found-messages-in-swagger-ui). The Hugo theme just delivers it as a dependency and _it_ generates the HTML. The `$ref` appears to be emitted from the javascript used by swagger ui. 

TL;DR: we can't fix this. The best we can do is hide the on-page error, which is sad from an engineering standpoint but not really user-facing.